### DISCUSSION TITLE
Skyline plugin

### DISCUSSION THREAD

**Discussion by wetlikej3** [Raytwo/ARCropolis#567] [Issues]
So, I'm not really the best when it comes to this stuff but usually you the new version comes out you update things but since the 13.0.2 came out I open it up and it says skyline plugin has panicked saying this:
![image](https://github.com/Raytwo/ARCropolis/assets/160173963/39140f37-8846-4c40-884f-07ef509324f2)

**Comment by ilikessbutoonlink** [comment]
clear out both romfs and exefs in atmosphere/contents/01006A800016E000 and delete arcropolis folder in ultimate, reinstall them, and see if that fixes the problem.
If it crash, don't install the nro plugins yet and see if it can boot up (obviously don't delete libarcropolis.nro). If it does boot up, one by one install these nro plugins.
Said plugin might not be compatible with whatever supporting plugin its using (many suggest to use the beta version of NRO Hook to fix issues)
Delete the arcropolis folder in ultimate and boot the game again. It will regenerate a fresh one.
You can do Fix Archive Bit in Hekate and see if that fixes the problem.
If problem persist, backup everything and reformat your sd card. Be sure that you change the format to FAT32. If you can't do that, you can convert NTFS to FAT32 using a 3rd party application that you can find on Google. Unfortunately, I'm still on exFAT but I never got a single sd card issue at all so I can't provide which is the best converter. Some suggest that you can change format using [Hekate](https://github.com/CTCaer/hekate/releases).
If that fails too, switch to another sd card. I use Sandisk for my sd card.
I got it from the gamebanana troubleshooting
